core/modules/replica/StatusRequest.h

```diff
@@ -58,7 +58,8 @@ namespace replica {
 //   Customizations for specific request types require dedicated policies
 // ========================================================================
 
-struct StatusReplicationRequestPolicy {
+class StatusReplicationRequestPolicy {
```

Comment:
The change is fine. I'm just curious about a reason for this change.

Reply:
clang / llvm warns on inconsistent use of {{class}} vs {{struct}}, so we should pick one in each case.

And easier, I think, to just start using {{class}} everywhere, even for POD?  That way you don't have to think about rules of what is POD and what is not, have people interpret differently and argue, remember to change/update code when appropriate, etc.